### PR TITLE
[CV-407] Lv Owner Map Screen

### PR DESCRIPTION
**Pull request by lo-vil**
ClickUp Issue: [Issue](https://app.clickup.com/t/86aztyxt5)

As seen on screenshot map component is generated with markers at locations of properties (not condo units).

This is only compatible with Android and IOS, not web. No map will be implemented for web.

When a user clicks on a pin, it brings them back to the properties list.

Screenshot 2024-02-29 at 1 28 06 AM

### CODE REVIEW

**Review by TheOneWithTheShot — DISMISSED** [review]

File: apps/native/screens/OwnerMapScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    useEffect(() => {
+        const fetchUserId = async () => {
+            // You'll see that userId is logged in console (just not in time for fetchData function)
+            await AsyncStorage.getItem('userUID')
+                .then((value) => {
+                    const data = JSON.parse(value);
+                    console.log('JSON data: ' + data)
+                    setUserId(data);
+                });
+        }
```
Comment (im-samwong) [inline apps/native/screens/OwnerMapScreen.tsx:24]: Nice job here with the asyncstorage, I think this is how everyone should be checking this the same way.

**Review by im-samwong — APPROVED** [review]
Great work.

**Comment by im-samwong** [comment]
Need to move files into components for now within native. Will need to restructure repo later.

**Comment by im-samwong** [comment]
Need to move files into components for now within native. Will need to restructure repo later.

**Review by Adissuu — DISMISSED** [review]
Great job creating that MapComponent for later use!
Just be careful for the merge conflict on the Login once all the branches merge.

**Review by XavierGuertin — DISMISSED** [review]

**Review by Adissuu — APPROVED** [review]
great

**Review by XavierGuertin — APPROVED** [review]

**Review by im-samwong — APPROVED** [review]
Good work. After long long long time.

**Pull request merged**